.gitignore

```diff
@@ -31,4 +31,6 @@ bld/
 [Oo]ut/
 msbuild.log
 msbuild.err
-msbuild.wrn
\ No newline at end of file
+msbuild.wrn
+
+local/db/sql
```

Comment:
Cheeky add: ignore Docker data directories when using `docker compose up` from the mono repo